teeio-validator/library/cxl_ide_test_lib/test_config/test_config_default.c

```diff
@@ -0,0 +1,54 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023-2024 Intel. All rights reserved.
+ *  License: BSD 3-Clause License.
+ **/
+
+#include <stdlib.h>
+#include <ctype.h>
+
+#include "hal/base.h"
+#include "hal/library/debuglib.h"
+
+#include "ide_test.h"
+#include "helperlib.h"
+#include "teeio_debug.h"
+#include "cxl_ide_lib.h"
+#include "cxl_ide_test_common.h"
+
+// test selective_ide and link_ide with default config
+bool cxl_ide_test_config_default_enable(void *test_context)
+{
+  return true;
+}
+
+bool cxl_ide_test_config_default_disable(void *test_context)
+{
+  return true;
+}
+
+bool cxl_ide_test_config_default_support(void *test_context)
+{
+  ide_common_test_config_context_t *config_context = (ide_common_test_config_context_t *)test_context;
+  TEEIO_ASSERT(config_context->signature == CONFIG_CONTEXT_SIGNATURE);
+
+  cxl_ide_test_group_context_t *group_context = config_context->group_context;
+  TEEIO_ASSERT(group_context->signature == GROUP_CONTEXT_SIGNATURE);
+
+  CXL_PRIV_DATA* rp_cxl_data = &group_context->upper_port.cxl_data;
+  CXL_PRIV_DATA* ep_cxl_data = &group_context->lower_port.cxl_data;
+
+  // TODO
```

Comment:
what is this TODO?

Reply:
This TODO shall be deleted, together with other un-used lines in the codes.